server/journeys/record-investigation/staff-involved/tests.test.ts

```diff
@@ -0,0 +1,139 @@
+import { Locals, Request } from 'express'
+import { v4 as uuidV4 } from 'uuid'
+import { agent as request } from 'supertest'
+import { getByRole, getByText } from '@testing-library/dom'
+import { appWithAllRoutes } from '../../../routes/testutils/appSetup'
+import testRequestCaptor, { TestRequestCaptured } from '../../../routes/testutils/testRequestCaptor'
+import createTestHtmlElement from '../../../routes/testutils/createTestHtmlElement'
+import { JourneyData } from '../../../@types/express'
+import { TEST_PRISONER } from '../../../routes/testutils/testConstants'
+
+const TEST_PATH = 'record-investigation/staff-involved'
+const uuid = uuidV4()
+const journeyDataMock = {
+  prisoner: TEST_PRISONER,
+  investigation: {},
+} as JourneyData
+
+let reqCaptured: TestRequestCaptured
+let requestCaptor: (req: Request) => void
+
+const app = (
+  {
+    journeyData,
+    validationErrors,
+  }: {
+    journeyData?: JourneyData
+    validationErrors?: Locals['validationErrors']
+  } = { journeyData: journeyDataMock, validationErrors: undefined },
+) => {
+  ;[reqCaptured, requestCaptor] = testRequestCaptor(journeyData, uuid)
```

Comment:
on this particular line, the semi-colon isn't needed.
hmpps typescript template chose to _not_ use semi-colon line ending, so any line starting with `(` or `[` has the risk of getting interpreted as a function call or index accessor from the previous line.

hence es-lint enforce a preceding `;` whenever a line start with `(` or `[` (regardless of what's in the previous line)

so it's a meaningless semi-colon that we're forced to add otherwise it will fail eslint check.